PGM/src/main/java/tc/oc/pgm/Config.java

```diff
@@ -119,6 +119,15 @@ public static boolean showProximity() {
         public static boolean preciseProgress() {
             return getConfiguration().getBoolean("scoreboard.precise-progress", false);
         }
+
+        public static boolean showIP() {
```

Comment:
we don't need this, the fact that "ScoreboardLines" is more than 0 is what makes it show or not